front/src/modules/command-menu/components/CommandMenuStyles.tsx

```diff
@@ -1,23 +1,22 @@
 import styled from '@emotion/styled';
 import { Command } from 'cmdk';
 
-import { useIsMobile } from '@/ui/utilities/responsive/hooks/useIsMobile';
-
-export const StyledDialog = styled(Command.Dialog)`
+export const StyledDialog = styled(Command.Dialog)<{ isMobile: boolean }>`
   background: ${({ theme }) => theme.background.primary};
   border-radius: ${({ theme }) => theme.border.radius.md};
   box-shadow: ${({ theme }) => theme.boxShadow.strong};
   font-family: ${({ theme }) => theme.font.family};
+  height: ${({ isMobile, theme }) =>
```

Comment:
the box shadow feels a bit strange and not consistent with other tabs. Maybe we should just leave height to 100% in that case?

Reply:
maybe your suggestion is z-index, but with z-index if the position is not relative can't handle events